squeasel.c

```diff
@@ -4215,7 +4220,22 @@ static int set_ssl_option(struct sq_context *ctx) {
     CRYPTO_set_id_callback(&ssl_id_callback);
   }
 
-  if ((ctx->ssl_ctx = SSL_CTX_new(SSLv23_server_method())) == NULL) {
+  char* ssl_version = ctx->config[SSL_VERSION];
+  int options = SSL_OP_NO_SSLv2 | SSL_OP_NO_SSLv3;
+  if (sq_strcasecmp(ssl_version, "tlsv1") == 0) {
+    // No-op - don't exclude any TLS protocols.
+#if OPENSSL_VERSION_NUMBER >= OPENSSL_VERSION_HAS_TLS_1_1
+  } else if (sq_strcasecmp(ssl_version, "tlsv1_1") == 0) {
```

Comment:
in Kudu we have a similar config but we use '.' instead of '_' in the string. ie this would be "tlsv1.1" instead of "tlsv1_1". Do you mind doing the same for consistency?